proto/net.proto

```diff
@@ -39,6 +39,14 @@ message Weights {
     optional float min_val = 1;
     optional float max_val = 2;
     optional bytes params = 3;
+    enum Encoding {
+      UNKNOWN = 0;
```

Comment:
nit: `UNKNOWN_ENCODING`.

(enums appear in parent namespace, so we need different UNKNOWN_* names for the case we ever have several enums)